pkgs/by-name/st/stalwart-mail/spam-filter.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "spam-filter";
+  version = "2.0.3";
+
+  src = fetchFromGitHub {
+    owner = "stalwartlabs";
+    repo = "spam-filter";
+    tag = "v${version}";
+    hash = "sha256-NhD/qUiGhgESwR2IOzAHfDATRlgWMcCktlktvVfDONk=";
+  };
+
+  # TODO Potentially retrieve / pre-fetch / store `https://` resources inside the file
+  buildPhase = ''
+    bash ./build.sh
+  '';
+
+  installPhase = ''
+    mkdir -p $out
+    cp spam-filter.toml $out/
+  '';
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  meta = {
+    description = "Spam Filter Rules";
+    homepage = "https://github.com/stalwartlabs/spam-filter";
+    changelog = "https://github.com/stalwartlabs/spam-filter/blob/${src.tag}/CHANGELOG.md";
+    license = lib.licenses.mit;
```

Comment:
On GitHub they have both an Apache as an MIT License. I'm not sure how you would pass that through the meta tag.

Reply:
You can use a list and put both licenses in there.

Reply:
Thank you :)